libs/application/core/src/types/Application.ts

```diff
@@ -22,4 +22,6 @@ export interface Application {
   attachments: object
   answers: FormValue
   externalData: ExternalData
+  name?: string
```

Comment:
so this won't be `MessageDescriptor | string` ?

Reply:
No, this is the application after it is returned from the backend, right ?

Reply:
yes, I was just a bit lost about where the actual message descriptor resides. but this is fine